packages/components/src/list-tile/style.scss

```diff
@@ -0,0 +1,29 @@
+.list-tile {
```

Comment:
How do you feel about using emotion instead of SCSS?

Reply:
We could, I was just following the pattern I see with other components in the folder.

Reply:
I see, this is in `packages`. I think it's a better idea to keep it the same structure there.